bld_server/src/endpoints/hist.rs

```diff
@@ -1,28 +1,38 @@
 use crate::extractors::User;
-use actix_web::{get, web::Data, HttpResponse, Responder};
+use actix_web::{
+    get,
+    web::{Data, Query},
+    HttpResponse, Responder,
+};
 use bld_core::database::pipeline_runs;
 use diesel::r2d2::{ConnectionManager, Pool};
 use diesel::sqlite::SqliteConnection;
 use tracing::info;
 
-#[get("/hist")]
+#[get("/hist/state/:")]
```

Comment:
Since we pass the state as a query parameter the url should be presented like 
```suggestion
#[get("/hist")]
```